incubator/model-avro/src/main/java/io/aklivity/zilla/runtime/model/avro/internal/AvroModelHandler.java

```diff
@@ -217,4 +244,13 @@ private int calculatePadding(
         }
         return padding;
     }
+
+    private int generateCRC32C(
+        String schemaText)
+    {
+        byte[] bytes = schemaText.getBytes();
```

Comment:
This line is going to allocate a new byte[] on every call to validate.
Let's instead store the actual schemaText rather than a crc32c hash to compare.